src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -53,6 +54,10 @@ function isPodiatry(state) {
   return getFormData(state).typeOfCareId === PODIATRY;
 }
 
+function getFacilityPageKey(state) {
+  return vaosFlatFacilityPage(state) ? 'vaFacilityV2' : 'vaFacility';
+}
```

Comment:
Helper function that will check the feature flag and then determine whether we should return the new or old VA facility page.  Below you'll see I replace all instances of `vaFacility` with this function.